pilot/cmd/pilot-agent/status/util/util.go

```diff
@@ -22,7 +22,7 @@ import (
 	"time"
 )
 
-const requestTimeout = time.Second * 2
+const requestTimeout = time.Second * 1 // Default readiness probe timeout.
```

Comment:
Why change this?

Reply:
nevermind I see the k8s timeout is 1s, I thought it was 2s. Makes sense to be the same I guess